easybuild/easyconfigs/p/Pillow/Pillow-3.4.2-foss-2016b-Python-3.5.2.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonPackage'
+
+name = 'Pillow'
+version = '3.4.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://pillow.readthedocs.org/'
+description = """Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.
+ PIL is the Python Imaging Library by Fredrik Lundh and Contributors."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+checksums = ['0ee9975c05602e755ff5000232e0335ba30d507f6261922a658ee11b1cec36d1']
+
+dependencies = [
+    ('Python', '3.5.2'),
+    ('libjpeg-turbo', '1.5.1'),
+    ('libpng', '1.6.26'),
+    ('zlib', '1.2.8'),
+    ('LibTIFF', '4.0.6'),
+    ('freetype', '2.7'),
+]
+
+options = {'modulename': 'PIL'}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = ["python -c 'import PIL'"]
```

Comment:
this is already being done by default, the line above specifying `PIL` tell EasyBuild to check with `import PIL` rather than `import Pillow`, so you can remove this line